test/colony.js

```diff
@@ -723,4 +733,77 @@ contract("Colony", addresses => {
       await checkErrorRevert(colony.claimPayout(taskId, MANAGER_ROLE, token.address, { from: OTHER }));
     });
   });
+
+  describe("when locking tokens", async () => {
```

Comment:
Take this out into its own test file. Token locking logic lives outside the `Colony` and ever outside the `ColonyNetwork`, similarly to the `ColonyNetworkAuction` and `ColonyNetworkMining`